README.md

```diff
@@ -1,10 +1,63 @@
 ## angular-filemanager
 File manager developed with AngularJS and Bootstrap by [Jonas Sciangula Street](https://github.com/joni2back)
 
+### use in your project
+
+#### 1) install and use
+1) bower install --save angular-filemanager
+2) include the dependencies in your project:
+
+    <!-- third party -->
+    <script src="/bower_components/angular-translate/angular-translate.min.js"></script>
+    <script src="/bower_components/angular-cookies/angular-cookies.min.js"></script>
+    
+    <!-- angular-filemanager -->
+    <link rel="stylesheet" href="/bower_components/angular-filemanager/dist/angular-filemanager.css">
+    <script src="/bower_components/angular-filemanager/dist/angular-filemanager.min.js"></script>
+    <script src="/bower_components/angular-filemanager/dist/cached-templates.js"></script>
+3) use the angular directive in your HTML:
+
+    <angular-file-manager></angular-file-manager>
+    
+#### 2) configure
+
+To configure the file manager you can simply add a config to your `config.js`:
+
+    angular.module('yourApp')
```

Comment:
When I use a third party dependecy I don't want to modify the original source, instead I must configure the default values in my angular app, so I've added these lines of code to see where the user who uses your package can configure `angular-filemanager` in their app config. It's necessary here to do this not in the `angular.module(...).config(...)` definition. Otherwise we couldn't access the config phase of your project. But this is still not perfect, I want to improve it more, but atm it works...